indexing-service/src/main/java/org/apache/druid/indexing/seekablestream/supervisor/SeekableStreamSupervisor.java

```diff
@@ -804,6 +805,15 @@ public SeekableStreamSupervisor(
               : Math.min(10, this.ioConfig.getTaskCount() * this.ioConfig.getReplicas()));
     }
 
+    if (spec.getIoConfig().getIdleConfig() != null) {
+      idleConfig = spec.getIoConfig().getIdleConfig();
```

Comment:
It's possible that users have only specified enabled/disabled at the supervisor level, but not the timeout value. Shouldn't we pick up the timeout value defined at the cluster level then?